sylvia-derive/src/multitest.rs

```diff
@@ -488,6 +488,10 @@ where
                         self.code_id
                     }
 
+                    pub fn code_info(&self) -> #sylvia ::cw_std::StdResult< #sylvia ::cw_std::CodeInfoResponse> {
```

Comment:
This should be behind feature flag as well (see the other review about instantiate2)